install_config/configuring_aws.adoc

```diff
@@ -18,6 +18,21 @@ volumes as persistent storage] for application data. After AWS is configured
 properly, some additional configurations will need to be completed on the
 OpenShift hosts.
 
+[IMPORTANT]
+====
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switched from using the *hostname* as the `*externalID*` (which would have
```

Comment:
@ahardin-rh Sorry, should have been more specific. Suggesting a word change:

s/switched/switches/

My question mark was because I wasn't totally sure if it mattered / was worth suggesting. Felt better, matching the tense of "tries" earlier in the sentence.